samples/sample-manifest.json

```diff
@@ -20,13 +20,24 @@
             "MaxActivations": 1,
             "Parameters": {
                 "Files": {
-                    "/tmp/defg.txt": "foo",
-                    "/tmp/jkl.mno": "bar"
+                    "/tmp/abc.txt": "helloworld",
+                    "/tmp/helloworld.sh": "#!/usr/bin/env bash\necho {{ raw .Secrets.hello }} {{ raw .Secrets.hello }}",
+                    "/tmp/defg.txt": {
+                        "encoding": "string",
+                        "data": "foo"
+                    },
+                    "/tmp/jkl.mno": {
+                        "encoding": "base64",
+                        "data": "YmFy"
+                    },
+                    "/tmp/enc.key": {
+                        "encoding": "bytes",
```

Comment:
what's the difference between `string` and `bytes`?

Reply:
`string` has some encoding. We could go with utf-8 for example. `bytes` are just raw bytes.
Basically, `bytes` tells Marblerun that it needs to handle templating differently.
As mentioned above we could also have a separate `templates` flag or smth. and only have `string` as an encoding type.

Reply:
We don't need bytes if we have the "noTemplates" flag, so this is obsolete.